lib/Ohmbrewer_Onewire.cpp

```diff
@@ -3,16 +3,18 @@
 #include "ds18x20.h"
 #include "onewire.h"
 
+
 /**
  * Constructors
  * @param busPin Digital pin that serves as the bus for all onewire
  * @param probeId Unique ID for the temperature probe [8] char array ID code
  */
 Ohmbrewer::Onewire::Onewire() : Ohmbrewer::Probe(){
-//    ow_setPin(D0);
+    Serial.println("Onewire");
+//    ow_setPin(D0);    //set globally
```

Comment:
Looking at this PR, I'm wondering if it would be a good idea to make a static function in Ohmbrewer::Onewire that just wraps ow_setPin(). It would mean we don't have to include onewire.h in the .ino file and it would be a little cleaner when we implement other Probe types. Something like Ohmbrewer::Onewire::init(int busProbe) or some such. Not strictly necessary for this PR though. Just throwing it out there.